components/compliance-service/migrations/migrations.go

```diff
@@ -0,0 +1,112 @@
+package migrations
+
+import (
+	"context"
+	"encoding/json"
+	"time"
+
+	"github.com/olivere/elastic/v7"
+	"github.com/sirupsen/logrus"
+)
+
+type Node struct {
+	NodeUUID  string `json:"node_uuid"`
+	EndTime   string `json:"end_time"`
+	DayLatest bool   `json:"day_latest"`
+}
+
+var setNodes map[string]Node
+
+func GetNodesDayLatestTrue(client *elastic.Client, ctx context.Context) ([]Node, error) {
+	var nodes []Node
+	termQueryDayLatest := elastic.NewTermQuery("day_latest", true)
```

Comment:
We need to check for daily_latest Flag as well. Only mark the records false, where both of them are true.